addons/web/static/src/views/view.xml

```diff
@@ -5,4 +5,32 @@
     <WithSearch t-props="withSearchProps" class="o_action o_view_controller"/>
   </t>
 
+  <t t-name="web.ReportViewMeasures" owl="1">
+    <Dropdown t-on-dropdown-item-selected="onMeasureSelected" togglerClass="'btn btn-primary'">
```

Comment:
t-on-dropdown-item-selected="onMeasureSelected" does not seem to be at the right place